app/controllers/HomeController.scala

```diff
@@ -5,16 +5,17 @@ import javax.inject._
 import com.typesafe.scalalogging.LazyLogging
 import play.api.mvc._
 
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
```

Comment:
from the thread name printed in log messages.

for details, in my owned project, before this change, my log is:

```
# Log format
# <pattern>%coloredLevel [%mdc{trackingId:--}] [%thread] %logger{15} - %message%n%xException{10}</pattern>

[info] [-] [scala-execution-context-global-80] play.api.Play - Application started (Dev)
[debug] [006bc4df-0310-4793-a88d-923dfde227d9] [play-dev-mode-akka.actor.default-dispatcher-2] c.l.TrackingFilter -  <:> trackingId generated: 006bc4df-0310-4793-a88d-923dfde227d9
[debug] [006bc4df-0310-4793-a88d-923dfde227d9] [application-akka.actor.default-dispatcher-2] - old context: null
[debug] [006bc4df-0310-4793-a88d-923dfde227d9] [application-akka.actor.default-dispatcher-2] - new context: {trackingId=006bc4df-0310-4793-a88d-923dfde227d9}
[warn] [-] [scala-execution-context-global-80] - ['token': 'fd'] | ['request': 'POST /api/auth'] | ['clientIp': '0:0:0:0:0:0:0:1'] <:> Valid token not found
```


Pay attention on the thread name `scala-execution-context-global-80`.

So 

* I guess I must imported the `scala.concurrent.ExecutionContext.Implicits.global` somewhere, and yes, I found by searching
* And I guess that the `MDCPropagatingDispatcher` is Akka Dispatcher, should not be related to the build-in `scala.concurrent.ExecutionContext.global`
* Also Play encouraged using the Dependency injection, so I need try to inject the `ExecutionContext`.

And then I change the code like this PR, and it works!